tools/tpp-opt/tpp-opt.cpp

```diff
@@ -30,6 +30,7 @@
 #include "TPP/Dialect/Xsmm/XsmmDialect.h"
 #include "TPP/PassBundles.h"
 #include "TPP/Passes.h"
+#include "TPP/Passes.h.inc"
```

Comment:
Nope. If you need this, something is really wrong elsewhere.

Reply:
This isn't necessary, I forgot to remove this.

Reply:
Done.